### DISCUSSION TITLE
Overriding and customising the standard Pega list view (Infinity 23)

### DISCUSSION THREAD

**Discussion by ozsaffer** [pegasystems/react-sdk#408] [Q&A]
I am currently investigating the DX component builder and I am looking to override the standard Pega list view component to add functionality to provide conditional styling and a thumbnail column. What is the best way to achieve this?

I have tried override the list view component, make changes and publish(received a success message). But my changes are not appearing in the app studio? I have successfully published custom widgets and field override so I assume my setup is correct.

**Comment by tumms2021389** [comment]
Hi @ozsaffer ,

Thank you for utilizing our React SDK components. I'd like to clarify your specific use case and whether the customization of the List View component is required in the React SDK portal or the Constellation portal.

If you intend to customize it within the React SDK, you can create an override for the ListView component and implement your modifications. These changes will be automatically reflected in the React SDK portal.

However, if your customization needs extend to both the React SDK portal and the Constellation Portal (App Studio), you'll need to make adjustments in both the custom-sdk and custom-constellation component folders. To modify the custom-constellation component, you can refer to the design.pega.com for information on the Constellation design system.

**Reply by ozsaffer** [reply]
Thanks for the reply,

I have tested overriding the list view in the React SDK portal and that is working, it looks like it is a reduced feature version, as compared to the Constellation portal ListView, is that correct?

I am to now trying override the ListView for the Constellation portal(preferred method) but I assume that this is not possible and that I need to create a new custom ListView component with my changes and then publish it?

Are there any other solutions to override styling in the Constellation portal ListView component, I have noticed that some of the columns have conditional formatting as show in the "Status" column(mapped from Work Status in sample MediaCo app). I am looking to add conditional formatting to the "DisplayDate"

![image](https://github.com/pegasystems/react-sdk/assets/355818/70655cd9-b902-4846-a0f3-31dcd13748ab)
